app/Http/Middleware/HandleInertiaRequests.php

```diff
@@ -55,6 +55,7 @@ public function share(Request $request): array
             'ziggy' => fn () => [
                 ...(new Ziggy())->toArray(),
                 'location' => $request->url(),
+                'query' => $request->query(),
```

Comment:
This lets us access the current query params on the server-side render in `usePageProps()`. If we don't do this, we only have access to them client-side using something like `window.location`, which could lead to hydration issues.